.eslintrc.js

```diff
@@ -1,25 +1,29 @@
+require('@rushstack/eslint-patch/modern-module-resolution')
+
 module.exports = {
   env: {
     browser: true,
     es2021: true,
     node: true,
   },
-  parser: 'vue-eslint-parser',
-  extends: ['eslint:recommended', 'plugin:vue/essential', 'plugin:@typescript-eslint/recommended'],
+  extends: [
+    'eslint:recommended',
+    'standard',
+    'plugin:vue/vue3-recommended',
+    'plugin:vue-scoped-css/vue3-recommended',
+    '@vue/eslint-config-typescript/recommended'
+  ],
   parserOptions: {
     ecmaVersion: 12,
-    parser: '@typescript-eslint/parser',
     sourceType: 'module',
   },
-  plugins: ['vue', '@typescript-eslint'],
   rules: {
-    'linebreak-style': ['error', 'unix'],
-    quotes: ['error', 'single'],
-    semi: ['error', 'never'],
-    'vue/multi-word-component-names': 'off',
     'comma-dangle': ['error', 'always-multiline'],
+    'linebreak-style': ['error', 'unix'],
     'no-multiple-empty-lines': 'error',
     'object-curly-spacing': ['error', 'always'],
-    'vue/no-v-model-argument': 'off',
+    'sort-imports': ['error', { ignoreDeclarationSort: true }],
+    'vue/block-order': ['error', { order: ['template', 'script', 'style'] }],
+    'vue/component-name-in-template-casing': ['error', 'kebab-case'],
```

Comment:
`parser` and `plugins` were not needed for everything to work correctly

rules `quotes` and `semi` are now part of the `standard` ruleset

`'vue/multi-word-component-names': 'off',` - we want it to be `ON` and it is as part of `'plugin:vue/vue3-recommended',`
`vue/no-v-model-argument` this was Vue 2 rule.